linux/map.jinja

```diff
@@ -185,3 +185,28 @@
          },
     },
 }, grain='os_family', merge=salt['pillar.get']('linux:storage')) %}
+
+
+{% set monitoring = salt['grains.filter_by']({
```

Comment:
You would need to refer `pillar.linux.get('monitoring')` in support meta.
It's ok to use map.jinja for it, just move the defaults there so you don't need to use .get('warn', sumthin)